fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCQueryBuilder.java

```diff
@@ -529,6 +527,7 @@ private SqlQueryData processReferenceParm(Class<?> resourceType, QueryParameter
             // Handle query parm representing this name/value pair construct:
             // <code>{name}:{Resource Type} = {resource-id}</code>
             if (queryParm.getModifier() != null && queryParm.getModifier().equals(Modifier.TYPE)) {
+                System.out.println(queryParm.getType());
```

Comment:
should it be a log statement?  or remove altogether?

Reply:
fixed  - it was meant  for debugging, and then I wanted to remove it.  Thanks for the eagle eyes